### DISCUSSION TITLE
Build @apply CSS files into separate file with CLI

### DISCUSSION THREAD

**Discussion by markmals** [unocss/unocss#1919] [Q&A]
I'm using the UnoCSS CLI with my Remix app and trying to compile utility classes (working well) as well as individual files which use the `@apply` directive.

Using [`@unocss/transformer-directives`](https://github.com/unocss/unocss/tree/main/packages/transformer-directives) with the UnoCSS CLI compiles my individual CSS files using the `@apply` directive in place, overwriting the use of `@apply` with plain CSS, instead of building these CSS files into the `--out-file` along with the utility classes, as I expected.

Is it possible to build CSS files which use `@apply` to the same file the utility classes are built to (or, even separate files) instead of replacing their contents in-place?

**Comment by codyzu** [comment]
This works with the windicss cli so I was also hoping it would work with uno. This would be very useful when integrating unocss into other static site generators (I'm using it with jekyll).
